src/components/Overlay/style.css

```diff
@@ -49,7 +41,34 @@
 }
 
 .circle:before {
-    content: "";
-    display: block;
-    padding-top: 125%;
+  content: "";
+  display: block;
+  padding-top: 125%;
+}*/
+
+/*.fullScreenOverlay {
+  @media (--small-viewport) {
+    .circle {
+        width: 70%;
+        border: 2.89px solid rgba(255,255,255,1);
+    }
+  }
+}
+*/
+
+.face {
```

Comment:
I am trying this SVG mask approach in order to reproduce the oval in the designs. Not sure if this has been tried in the past though, but proposing it just in case.